### DISCUSSION TITLE
[Aug 2024] Gem Development & Maintenance Status

### DISCUSSION THREAD

**Discussion by sofie-c** [Gusto/apollo-federation-ruby#280] [Announcements]
Hi everyone,

Wanted to post on behalf of the GraphQL team at Gusto (cc @slauppy, @simoncoffin, @sethc2) that we are still operating in deprioritization mode. 

As @simoncoffin so eloquently stated in #267:

>For now, we've decided our maintenance of this gem is unlikely to extend beyond simple changes, such as bug patches, dependency bumps, and very minor tweaks. More significant changes, such as breaking changes, new features, or alterations to fundamental internal logic will be deprioritized.

We are still maintaining this stance as we work through some of our larger internal initiatives.

> If you have an urgent need for a change to go through, we recommend you fork. We'll be more than happy to work with the community on merging their updates in the future. 

We still recommend this and still commit to working with folx in the future to merge in updates. 

>We anticipate being able to dedicate more resources to this gem in May 2024.

As of now, we are going to revisit this in January 2025. Thank you for your inquiries!

**Comment by palexvs** [comment]
Hi @sofie-c , @tubaxenor 
Really appreciate your work!
I saw some activities recently, does it mean you are back to maintain the gem?
I hope somebody could take a look on https://github.com/Gusto/apollo-federation-ruby/issues/276 . There are also a few opened PR to fix the issue

Thank you!
